jtstoolkit/src/main/java/ec/tstoolkit/modelling/arima/PreprocessingModel.java

```diff
@@ -567,7 +567,7 @@ public TsData forecast(int nf, boolean transformed) {
         TsData c = regressionEffect(f.getDomain());
         TsData r = TsData.add(f, c);
 
-        if (!transformed) {

```

Comment:
The routine should not be changed. The "transformed" parameter should be interpreted as follows: if you want to get the forecasts corresponding to the original series (untransformed), set "transformed" to "false"; if you want to get the forecasts corresponding to the transformed (in logs...) series, set "transformed" to "true". 
In the X11 package, the A1a table needs a call to "model.forecast(..., **false**)